api/README.md

```diff
@@ -186,5 +215,26 @@ curl localhost:5000/verify -F 'file1=@../data/renditions/1080p/0fIdY5IAnhY_60.mp
 ```
 #### Response
 ```
-{"model":"http://storage.googleapis.com/verification-models/verification-metamodel-fps2.tar.xz","orchestrator_id":"foo","results":[{"audio_available":false,"fps":60.0,"height":720,"ocsvm_dist":0.028662416537303254,"ssim_pred":0.9728838728836663,"tamper":0,"tamper_sl":0,"tamper_ul":1,"uri":"/tmp/d0424e5c79c9401d893d6f2b8e87dfc2/720_0fIdY5IAnhY_60.mp4","video_available":true,"width":1280}],"source":"/tmp/d0424e5c79c9401d893d6f2b8e87dfc2/1080_0fIdY5IAnhY_60.mp4"}
+{
+   "model":"http://storage.googleapis.com/verification-models/verification-metamodel-fps2.tar.xz",
```

Comment:
Can we fix the indentation here? Looks like brackets are misaligned which makes it a bit harder to visually parse the start/end of nested arrays/objects.